src/stories/containers/finances-overview/Components/ExpensesChart/ExpensesChart.tsx

```diff
@@ -47,14 +49,15 @@ const ExpensesChart: React.FC<Props> = ({ newActual, newDiscontinued, newPredict
         align: 'center',
         fontFamily: 'Inter,san-serif',
         fontWeight: 400,
-        fontSize: 9,
-        lineHeight: 11,
+        fontSize: upTable ? 12 : 9,
+        height: upTable ? 15 : 11,
         baseline: 'top',
         interval: 0,
       },
     },
     yAxis: {
       axisLabel: {
+        margin: upTable ? 16 : 7,
         // eslint-disable-next-line spellcheck/spell-checker
```

Comment:
This this `eslint-disable` required? Maybe if `formatted` is being detected as a typo it should be added to the dictionary